packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGPanZoom/SVGPanZoom.spec.js

```diff
@@ -50,18 +50,40 @@ describe('Components > SVGPanZoom', function () {
     expect(viewBox).to.equal('0 0 400 200')
   })
 
-  it('should enable horizontal panning', function () {
-    onPan(-1, 0)
-    wrapper.update()
-    const viewBox = wrapper.find('svg').prop('viewBox')
-    expect(viewBox).to.equal('10 0 400 200')
-  })
+  describe('panning', function () {
+    describe('left', function () {
+      it('should move the viewbox left', function () {
+        onPan(-1, 0)
+        wrapper.update()
+        const viewBox = wrapper.find('svg').prop('viewBox')
+        expect(viewBox).to.equal('-10 0 400 200')
+      })
+    })
 
-  it('should enable vertical panning', function () {
-    onPan(0, -1)
-    wrapper.update()
-    const viewBox = wrapper.find('svg').prop('viewBox')
-    expect(viewBox).to.equal('0 -10 400 200')
+    describe('right', function () {
+      it('should move the viewbox right', function () {
+        onPan(1, 0)
+        wrapper.update()
+        const viewBox = wrapper.find('svg').prop('viewBox')
+        expect(viewBox).to.equal('10 0 400 200')
+      })
+    })
+    describe('up', function () {
+      it('should move the viewbox up', function () {
+        onPan(0, -1)
+        wrapper.update()
+        const viewBox = wrapper.find('svg').prop('viewBox')
+        expect(viewBox).to.equal('0 -10 400 200')
+      })
+    })
+    describe('down', function () {
+      it('should move the viewbox down', function () {
+        onPan(0, 1)
+        wrapper.update()
+        const viewBox = wrapper.find('svg').prop('viewBox')
+        expect(viewBox).to.equal('0 10 400 200')
+      })
```

Comment:
Should each of these have expectations on what the viewBox was previously?

Reply:
Yes! Good catch.